src/main/java/org/osc/sdk/controller/api/SdnRedirectionApi.java

```diff
@@ -20,20 +20,34 @@
 
 import org.osc.sdk.controller.FailurePolicyType;
 import org.osc.sdk.controller.TagEncapsulationType;
+import org.osc.sdk.controller.element.Element;
 import org.osc.sdk.controller.element.InspectionHookElement;
 import org.osc.sdk.controller.element.InspectionPortElement;
 import org.osc.sdk.controller.element.NetworkElement;
 import org.osc.sdk.controller.exception.NetworkPortNotFoundException;
 import org.osgi.annotation.versioning.ConsumerType;
 
 /**
- * This interface represents API's required to implement traffic redirection
+ * <p>
+ * This interface represents API's required to implement traffic redirection.
+ *
+ * In case of SFC:
+ * <ul>
+ * <li>{@link SdnRedirectionApi#registerNetworkElement(List)} is used to register a port chain
+ * <li>{@link SdnRedirectionApi#updateNetworkElement(NetworkElement, List)} is used to update a port chain
+ * <li>{@link SdnRedirectionApi#deleteNetworkElement(NetworkElement)} is used to delete a port chain
+ * <li>{@link SdnRedirectionApi#installInspectionHook(List, InspectionPortElement, Long, TagEncapsulationType, Long, FailurePolicyType)} is used to create a flow classifier
+ * <li>{@link SdnRedirectionApi#removeInspectionHook(List, InspectionPortElement)}} is used to remove the list of flow classifier attached to a port chain
+ * </ul>
  */
 @ConsumerType
 public interface SdnRedirectionApi extends AutoCloseable {
 
     /**
+     * <p>
      * Creates a network element with a list of child elements on SDN controller
+     * In case of SFC, the network elements are the port pair group and return element is the port chain
```

Comment:
In case of SFC, the provided network element is the port pair group and the returned element is the port chain